src/components/DimensionMenu.js

```diff
@@ -310,7 +251,6 @@ DimensionMenu.propTypes = {
     classes: PropTypes.object,
     currentAxisId: PropTypes.string,
     dimensionId: PropTypes.string,
-    dualAxisItemHandler: PropTypes.func,
```

Comment:
Since `dualAxisItemHandler` was removed I guess this counts as a breaking change? @janhenrikoverland  @amcgee